### DISCUSSION TITLE
How to use Error handling correctly?

### DISCUSSION THREAD

**Discussion by codefln** [amannn/next-intl#1049] [General]
I'm using NEXT 14. And I want to get the fallback value for non-existent locale keys. According to the documentation: https://next-intl-docs.vercel.app/docs/usage/configuration#error-handling

But when I add getMessageFallback to my getRequestConfig I get an error

TypeError: createContext only works in Client Components. Add the "use client" directive at the top of the file to use it. Read more: https://nextjs.org/docs/messages/context-in-server-component

**Comment by amannn** [comment]
Are you importing something for the `getMessageFallback` implementation that (indirectly) imports `createContext` somewhere? Generally, `getMessageFallback` works in `i18n.ts` and doesn't rely on React Context.

**Comment by dotto95** [comment]
Hi, I have the same problems, I also posted on stackoverflow.

https://stackoverflow.com/questions/79071334/next-intl-getmessagefallback-in-client-component-throws-functions-cannot-be-pas

Any news about?
